server/key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Is this intended here?

Reply:
Oh committed by accident, but it was just an example anyway so nothing is lost